src/App.jsx

```diff
@@ -1,5 +1,12 @@
+import SideBarNav from "./components/side-bar-nav/SideBarNav";
+
 function App() {
-  return <h1>Happy Coding 🚀</h1>;
+	return (
```

Comment:
I think your tab space on your linter has 3 spaces rather than 2. Can you change the rule maybe? the tab space seems excessive